src/Compilers/Core/RebuildTest/CSharpRebuildTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using BuildValidator;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Rebuild.UnitTests
+{
+    public class CSharpRebuildTests : CSharpTestBase
```

Comment:
Why the new class here vs. adding onto the existing test case?

Reply:
I felt like "round tripping various compilation options" was pretty well delineated as a category of tests from "using various C# language features"

Reply:
What type of test do you imagine won't involve a round trip at the end though? That in many ways is the ultimate test of the code here  😄

Reply:
They all will test that round tripping works. But some tests are primarily about passing "unusual" compilation options while other tests are primarily about using "unusual" language features.

That said, if you want me to put this in the existing file, just say so and I will.

Reply:
I'll let you decide.